.github/workflows/set-announce-url.yml

```diff
@@ -0,0 +1,62 @@
+name: Set Announcement URL
+run-name: Set Announcement URL - ${{ inputs.announcement-url }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      announcement-url:
+        description: 'The URL of the announcement'
+        type: string
+        required: true
+
+jobs:
+  set-announce-url:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    steps:
+      - name: Validate inputs
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const announcementUrl = '${{ inputs.announcement-url }}';
+
+            if (!announcementUrl.includes('http')) {
+              throw new Error(`announcement-url must be a valid URL, got: ${announcementUrl}`);
+            }
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_S3_RELEASE_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_S3_RELEASE_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Update announcement url
+        uses: actions/github-script@v6
+        id: announce_url
+        with:
+          result-encoding: string
+          script: |
+            // TODO: move this to the release/ directory when it exists
+            const fs = require('fs');
+
+            const announcementUrl = '${{ inputs.announcement-url }}';
+
+            const OSSversionInfo = await fetch(`http://${{ vars.AWS_S3_PUBLIC_BUCKET }}.s3.us-east-1.amazonaws.com/version-info.json`).then(r => r.json());
+            const EEversionInfo = await fetch(`http://${{ vars.AWS_S3_PUBLIC_BUCKET }}.s3.us-east-1.amazonaws.com/version-info-ee.json`).then(r => r.json());
```

Comment:
Why do we use `fetch` here when we can do it using the provided `aws` cli tool, like in the "Upload" step?
This also has a hard-coded region which might change.

Reply:
1. I updated the hard-coded region to use a variable. 👍 
2. I think it makes sense to use fetch because it ensures we're getting a publicly accessible object, and I think it's simpler because we can fetch and parse all at once instead of download to disk, load into js from the filesystem, then parse.